File: frontend/src/app/types/report.type.ts
type ReportType = 'not_exist' | 'wrong_location' | 'inconsistent_photo';

export type ReportInterface = {
  id: number;
  message: string;
  report_type: ReportType;
  user_id: number;
};

export type CreateReportDTO = Omit<ReportInterface, 'id'>;

File: backend/Enums/ReportType.cs
using System.Runtime.Serialization;

namespace Backend.Enums;

public enum ReportType
{
  [EnumMember(Value = "not_exist")]
  NotExist,
  [EnumMember(Value = "wrong_location")]
  WrongLocation,
  [EnumMember(Value = "inconsistent_photo")]
  InconsistentPhoto,
}
